.github/workflows/ci.yaml

```diff
@@ -47,24 +47,10 @@ jobs:
           ini-values: memory_limit=512M, xdebug.mode=off
           tools: composer:v2.1
 
-      - name: Get composer cache directory
-        id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
-
-      - name: Cache dependencies
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.composer-cache.outputs.dir }}
-          key: composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('composer.*') }}
-          restore-keys: |
-            composer-${{ runner.os }}-${{ matrix.php-version }}-${{ hashFiles('composer.*') }}-
-            composer-${{ runner.os }}-${{ matrix.php-version }}-
-            composer-${{ runner.os }}-
-            composer-
-
       - name: Install dependencies
-        run: |
-          composer install --no-interaction --prefer-dist --no-progress
+        uses: ramsey/composer-install@v2
+        with:
+            dependency-versions: locked
 
       - name: Run tests
         shell: bash
```

Comment:
Didn't change it for bellow: I admit I'm really not comfortable with it yet (the tests executed)